### PR TITLE
treewide: fix typos

### PR DESCRIPTION
**Pull request by torstehu**

### CODE REVIEW

**Comment by deathbybandaid** [comment]
I'm not familiar with treewide, had to google it, but this looks fairly thorough.

Crazy how many typos it found though!

**Comment by Exirel** [comment]
I love this. 👍 
Thank you very much.

File: sopel/tools/calculation.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def guarded_pow(left, right):
     elif not isinstance(right, numbers.Integral):
         pass
     elif pow_complexity(left, right) < 0.5:
-        # Value 0.5 is arbitary and based on a estimated runtime of 0.5s
+        # Value 0.5 is arbitrary and based on a estimated runtime of 0.5s
```
Comment (dgw) [inline sopel/tools/calculation.py]: Can we fix this too as long as the line is being touched anyway?

```suggestion
        # Value 0.5 is arbitrary and based on an estimated runtime of 0.5s
```

Comment (torstehu) [reply sopel/tools/calculation.py]: Will fix

**Pull request merged**